tests/specs/file-extensions.js

```diff
@@ -0,0 +1,37 @@
+var assert = require('assert');
+var path = require('path');
+var utils = require('../testing-utils');
+var browserify = utils.browserify;
+
+function testDirPath(testName) {
+  return path.join(__dirname, testName);
+}
+
+describe('file extension configuration', function() {
+
+  describe('extension whitelist', function() {
+
+    it('rejects invalid extensions', function(done) {
+      var stream = browserify({
+        root: testDirPath('invalid-extension')
+      });
+      stream.on('error', function(err) {
+        assert.ok(err);
+        done();
+      });
+    });
+
+    it('does something else', function(done) {
+      browserify({
+        root: testDirPath('default-extension')
+      }, function(err, tmpl) {
+        console.error(err);
+        assert.equal(err, null, 'no error was thrown');
```

Comment:
but when I try to process it, for some reason it errors out:
```
/Users/kadam/gh/kadamwhite/combynify/tests/specs/default-extension/template.cmbn:1
I am a .cmbn template
  ^
ParseError: Unexpected token
```
so the parsing isn't working right; not sure what I'm doing wrong.